lib/puppetserver/ca/local_certificate_authority.rb

```diff
@@ -73,33 +82,68 @@ def create_master_cert(ca_key, ca_cert)
                                               @settings[:hostpubkey])
         if master_key
           master_csr = @host.create_csr(name: @settings[:certname], key: master_key)
-          master_cert = sign_master_cert(ca_key, ca_cert, master_csr)
+          alt_names =
+            if @settings[:subject_alt_names].empty?
```

Comment:
I thought we decided not to do assignment of from non-one-line-ternary-conditionals somewhere else?

Reply:
Yeah I personally prefer variable assignment in both branches, this was just ripped directly from another part of the code. I'll update.